src/UiSlider.vue

```diff
@@ -543,7 +543,7 @@ $ui-slider-marker-size                      : rem(36px);
     left: 0;
     position: absolute;
     text-align: center;
-    top: rem(4px);
+    line-height: 2.3;
```

Comment:
Perhaps it'd be better to reset the line-height to 1 and use `top` to set a proper alignment? That way we don't have to wonder where the number `2.3` came from.